extra/utils/permissions.js

```diff
@@ -0,0 +1,26 @@
+const permissions = {
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+    }
+    }
+    function haspermission(modulename,role,permissiontype)
+    {
+        let roles=modulename[permissiontype];
```

Comment:
give spaces before and after equal to sign

Reply:
Done